File: infren/MP3_player/app/src/main/java/com/sy/example/mp3_player/MainActivity.java
package com.sy.example.mp3_player;

import androidx.appcompat.app.AppCompatActivity;

import android.media.MediaPlayer;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    TextView tv;
    MediaPlayer mp;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        mp=MediaPlayer.create(this, R.raw.wakeup);
        mp.setLooping(false);

        tv=this.findViewById(R.id.playbutton);
        tv.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                if(!mp.isPlaying()){
                    mp.start();
                    tv.setText("Stop");
                }
                else{
                    mp.pause();
                    tv.setText("Start");
                }
            }
        });
    }
}

File: Calculator/app/src/main/java/com/sy/example/calculator/MainActivity.kt
package com.sy.example.calculator

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import com.google.android.material.floatingactionbutton.FloatingActionButton

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val calculation=findViewById<TextView>(R.id.calculation)

        findViewById<Button>(R.id.button0).setOnClickListener{
            calculation.text=button0.text
        }
    }
}